scripts/rootpy-add-include-path

```diff
@@ -0,0 +1,23 @@
+#! /bin/env python
```

Comment:
Instead of adding a new script, could we add a new subparser in the main rootpy script?

```python
parser_include = subparsers.add_parser('include', description='blah')
parser_include.add_argument('paths', nargs='+', help='absolute include paths')
parser_include.set_defaults(op=add_include_path)
etc...
```